MDANSE_GUI/Src/MDANSE_GUI/Tabs/Visualisers/Action.py

```diff
@@ -410,14 +411,30 @@ def show_output_prediction(self):
     @Slot()
     def allow_execution(self):
         allow = True
+        has_warning = False
         for widget in self._widgets:
             if not widget._configurator.valid:
                 allow = False
+                widget.mark_error(widget._configurator.error_status, silent=True)
+            if widget.has_warning:
+                has_warning = True
```

Comment:
```suggestion
            has_warning = has_warning or widget.has_warning
```
Same meaning, style choice, no preference really.